src/api/services/TransactionAPI.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import querystring = require("querystring");
+import { CONFIG } from "../../Configuration";
+import { ITransactionListResponse } from "../types/TransactionResponse";
+
+// Transaction Service for PagoPA communications
+export class TransactionAPI {
+  // Retrieve transactions
+  public static getTransactionListResponse(
+    res: express.Response,
+    errorCallback: (response: express.Response, errorMsg: string) => void,
+    successCallback: (
+      response: express.Response,
+      transactionList: ITransactionListResponse
+    ) => void,
+    apiRequestToken: string,
+    id?: number,
+    start?: number,
+    size?: number
+  ): void {
+    let url; // tslint:disable-line
+    if (id !== undefined) {
```

Comment:
I suggest to split this method in two distinct methods (`getTransaction `and `getTransactions`): one that takes the id and the other one that does not take it (url can be a constant).